### PR TITLE
Rendertime: Admin configuration added

### PR DESCRIPTION
**Pull request by annando**
The addon is now easier to configure. This will help in cases like https://github.com/friendica/friendica/issues/11089

### CODE REVIEW

File: rendertime/rendertime.php [hunk 1]
```diff
@@ -8,15 +8,36 @@
...
 
 function rendertime_init_1(&$a) {
 }
 
+function rendertime_addon_admin(&$a, &$o) {
+	$t = Renderer::getMarkupTemplate("admin.tpl", "addon/rendertime/");
+
+	$o = Renderer::replaceMacros($t, [
+		'$submit' => DI::l10n()->t('Save Settings'),
+		'$callstack' => ['callstack', DI::l10n()->t('Show callstack'), DI::config()->get('rendertime', 'callstack'), 'Show detailed performance measures in the callstack. When deactivated, only the summary will be displayed.'],
```
Comment (nupplaphil) [inline rendertime/rendertime.php]:
```suggestion
		'$callstack' => ['callstack', DI::l10n()->t('Show callstack'), DI::config()->get('rendertime', 'callstack'), DI::l10n()->t('Show detailed performance measures in the callstack. When deactivated, only the summary will be displayed.')],
```

File: rendertime/rendertime.php [hunk 2]
```diff
@@ -8,15 +8,36 @@
...
+		'$minimal_time' => ['minimal_time', DI::l10n()->t('Minimal time'), DI::config()->get('rendertime', 'minimal_time'), 'Minimal time that an activity needs to be listed in the callstack.'],
```
Comment (nupplaphil) [inline rendertime/rendertime.php]:
```suggestion
		'$minimal_time' => ['minimal_time', DI::l10n()->t('Minimal time'), DI::config()->get('rendertime', 'minimal_time'), DI::l10n()->t('Minimal time that an activity needs to be listed in the callstack.')],
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]
Translating the description as well (needs `messages.po` recreation again ^^)

**Review by MrPetovan — APPROVED** [review]
Please address @nupplaphil 's review.

**Review by nupplaphil — APPROVED** [review]

**Pull request merged**